core/test/base/range_accessors.cpp

```diff
@@ -164,248 +135,4 @@ TEST_F(RowMajorAccessor, CanAssignSubranges)
 }
 
 
-class RowMajorAccessor3d : public ::testing::Test {
-protected:
-    using span = gko::span;
-    static constexpr gko::size_type dimensionality{3};
-
-    using row_major_int_range =
-        gko::range<gko::accessor::row_major<int, dimensionality>>;
-
-    // clang-format off
-    int data[2 * 3 * 4]{
-        1, 2, -1, 11,
-        3, 4, -2, 12,
-        5, 6, -3, 13,
-
-        21, 22, 23, 24,
-        25, 26, 27, 28,
-        29, 30, 31, 32
-    };
-    // clang-format on
-    const gko::dim<dimensionality> dim1{2, 3, 4};
-    const gko::dim<dimensionality> dim2{2, 2, 3};
-    row_major_int_range default_r{data, dim1};
-    row_major_int_range custom_r{
-        data, dim2,
-        std::array<const gko::size_type, dimensionality - 1>{12, 4}};
-};
-
-
-TEST_F(RowMajorAccessor3d, CanAccessData)
-{
-    EXPECT_EQ(default_r(0, 0, 0), 1);
-    EXPECT_EQ(custom_r(0, 0, 0), 1);
-    EXPECT_EQ(default_r(0, 1, 0), 3);
-    EXPECT_EQ(custom_r(0, 1, 0), 3);
-    EXPECT_EQ(default_r(0, 1, 3), 12);
-    EXPECT_EQ(default_r(0, 2, 2), -3);
-    EXPECT_EQ(default_r(1, 2, 1), 30);
-    EXPECT_EQ(default_r(1, 2, 2), 31);
-    EXPECT_EQ(default_r(1, 2, 3), 32);
-}
-
-
-TEST_F(RowMajorAccessor3d, CanWriteData)
-{
-    default_r(0, 0, 0) = 4;
-    custom_r(1, 1, 1) = 100;
-
-    EXPECT_EQ(default_r(0, 0, 0), 4);
-    EXPECT_EQ(custom_r(0, 0, 0), 4);
-    EXPECT_EQ(default_r(1, 1, 1), 100);
-    EXPECT_EQ(custom_r(1, 1, 1), 100);
-}
-
-
-TEST_F(RowMajorAccessor3d, CanCreateSubrange)
-{
-    auto subr = custom_r(span{0, 2}, span{1, 2}, span{1, 3});
-
-    EXPECT_EQ(subr(0, 0, 0), 4);
-    EXPECT_EQ(subr(0, 0, 1), -2);
-    EXPECT_EQ(subr(1, 0, 0), 26);
-    EXPECT_EQ(subr(1, 0, 1), 27);
-}
-
-
-TEST_F(RowMajorAccessor3d, CanCreateRowVector)
-{
-    auto subr = default_r(1u, 2u, span{0, 2});
-
-    EXPECT_EQ(subr(0, 0, 0), 29);
-    EXPECT_EQ(subr(0, 0, 1), 30);
-}
-
-
-TEST_F(RowMajorAccessor3d, CanCreateColumnVector)
-{
-    auto subr = default_r(span{0u, 2u}, 1u, 3u);
-
-    EXPECT_EQ(subr(0, 0, 0), 12);
-    EXPECT_EQ(subr(1, 0, 0), 28);
-}
-
-
-TEST_F(RowMajorAccessor3d, CanAssignValues)
-{
-    default_r(1, 1, 1) = default_r(0, 0, 0);
-
-    EXPECT_EQ(data[17], 1);
-}
-
-
-TEST_F(RowMajorAccessor3d, CanAssignSubranges)
```

Comment:
Because I removed the `copy_from` from the `gko::acc::range` (and I forgot to remove the `copy_from` from the Accessor itself). Thanks for hinting me on that.